docs/source/hotregion.rst

```diff
@@ -8,6 +8,20 @@ HotRegion
 Instances of :class:`~.HotRegion.HotRegion` are objects representing radiatively
 intense regions of the source photosphere.
 
+.. figure:: images/super_cede_omit.png
+   :width: 600
+
+Representation of the class instantiation options for the parameters `omit` and `cede` described below. 
+The image also shows the prefixes used to describe the parameters within a HotRegion object.
+
+.. figure:: images/HotRegion_parameters.png
+   :width: 600
+
+Representation of the parameters describing the HotRegion object, when `omit` or `cede` are set to ``True``.
```

Comment:
`Representation of a ``HotRegion`` instance configuration, when ``omit`` or ``cede`` are set to ``True``.
 A description of these parameters is also presented below. 
 If ``is_antiphased = True``, the parameter ``phase_shift`` is added to :math:`\pi` radians.
 For example, if ``phase_shift = 0``, the omission or the superseding region (respectively on the left and right side of the image) is in anti-phase with the observer meridian (i.e., its azimuth is :math:`\pi`).`

Reply:
hi @serenavinciguerra89 , I made some edits to your caption. Can you please check and push the edits?